core/domain/topic_domain.py

```diff
@@ -1258,14 +1262,18 @@ def update_url_fragment(self, new_url_fragment):
         """
         self.url_fragment = new_url_fragment
 
-    def update_thumbnail_filename(self, new_thumbnail_filename):
+    def update_thumbnail_filename_and_size_in_bytes(
```

Comment:
Keep the name as-is, figure out the size within this function.

Reply:
Done